src/main/c/sslcontext.c

```diff
@@ -1124,7 +1124,7 @@ TCN_IMPLEMENT_CALL(void, SSLContext, setAlpnProtos)(TCN_STDARGS, jlong ctx, jobj
         TCN_ASSERT(ctx != 0);
         UNREFERENCED(o);
 
-        if (initProtocols(e, c->pool, &c->alpn_proto_data, &c->alpn_proto_len, alpn_protos) == 0) {
+        if (initProtocols(e, c, &c->alpn_proto_data, &c->alpn_proto_len, alpn_protos) == 0) {
```

Comment:
this is a bit scary....maybe we should make compiler warning into errors...

Luckily it looks like this parameter wasn't used in `initProtocols`...should we just remove it from the signature?